build.sbt

```diff
@@ -30,9 +30,10 @@ lazy val redis =
     .settings(stdSettings("zio-redis"))
     .settings(buildInfoSettings("zio.redis"))
     .settings(
+      addCompilerPlugin("org.typelevel" % "kind-projector" % "0.13.2" cross CrossVersion.full),
```

Comment:
Please don't depend on kind-projector.

Reply:
Why? It is a part of scala 3 🤔

Reply:
It isn't the part of Scala 3. Scala 3 has a compatible (improved) type lambdas that are compatible with it. However, since we're in the 2.x realm, I'd prefer if we reduce the dependencies as much as possible.

Reply:
I thought it would be easier for us to migrate and handle 2 scala version with this plugin https://docs.scala-lang.org/scala3/guides/migration/plugin-kind-projector.html, but ok i get rid of it

Reply:
fixed